applications/StructuralMechanicsApplication/custom_elements/truss_element_linear_3D2N.cpp

```diff
@@ -189,20 +189,20 @@ void TrussElementLinear3D2N::WriteTransformationCoordinates(
     BoundedVector<double, TrussElement3D2N::msLocalSize>
     & rReferenceCoordinates)
 {
-    KRATOS_TRY;
+    KRATOS_TRY
     rReferenceCoordinates = ZeroVector(msLocalSize);
     rReferenceCoordinates[0] = GetGeometry()[0].X0();
     rReferenceCoordinates[1] = GetGeometry()[0].Y0();
     rReferenceCoordinates[2] = GetGeometry()[0].Z0();
     rReferenceCoordinates[3] = GetGeometry()[1].X0();
     rReferenceCoordinates[4] = GetGeometry()[1].Y0();

```

Comment:
you can use `const auto& r_initial_coords = GetGeometry()[0].InitialPosition();`